src/main/java/com/openlattice/hazelcast/HazelcastReliableTopic.kt

```diff
@@ -0,0 +1,9 @@
+package com.openlattice.hazelcast
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+enum class HazelcastReliableTopic {
```

Comment:
What is the purpose of this?

Reply:
Is an enum for reliable topics. I created one and then removed it so it has no reason to exist anymore, but left it in just in case.